src/renderer/views/pools/PendingPools.tsx

```diff
@@ -104,23 +105,28 @@ export const PendingPools: React.FC = (): JSX.Element => {
 
   const setFilter = useCallback((oFilter: O.Option<PoolFilter>) => setPoolsFilter(POOLS_KEY, oFilter), [setPoolsFilter])
 
-  const lastblock = useMemo(() => RD.toNullable(thorchainLastblockRD), [thorchainLastblockRD])
-  const constants = useMemo(() => RD.toNullable(thorchainConstantsRD), [thorchainConstantsRD])
-
   const renderBlockLeftColumn = useCallback(
     (_: string, record: PoolTableRowData) => {
       const { deepest, pool } = record
 
-      const blocksLeft =
-        lastblock && constants ? getBlocksLeftForPendingPoolAsString(constants, lastblock, pool.asset) : ''
+      const oLastblockItems = RD.toOption(thorchainLastblockRD)
+      const oConstants = RD.toOption(thorchainConstantsRD)
+
+      const blocksLeft = FP.pipe(
+        sequenceTOption(oLastblockItems, oConstants),
+        O.map(([lastblockItems, constants]) =>
+          getBlocksLeftForPendingPoolAsString(constants, lastblockItems, pool.target)
+        ),
+        O.getOrElse(() => '')
```

Comment:
nitpick: no need to convert `thorchainLastblockRD` and `thorchainConstantsRD` to the `Option`. we can work with `RD` directly